tests/syntax_error_in_code/input/two_fer.rb

```diff
@@ -0,0 +1,2 @@
+module TwoFer
+end,A stray comma
```

Comment:
Should we put an inline Rupocopy directive to ignore this rule so that this test is not ran for this line?  We can also specify it in a supplied temporary config file.

The inline or at least in file, directive will be less of a maintenance burden (No need to remember, since the reminder is in this file).

Reply:
I don't think we do, this is just a test case for the smoke tests, not actual Ruby code we'll be editing.